database/migrations/2018_11_01_200000_create_products_table.php

```diff
@@ -26,7 +26,12 @@ public function up()
             $table->string('name');
             $table->string('part_number');
             $table->string('internal_code')->nullable()->unique();
+
+            $table->unsignedInteger('packaging_unit_id')->index()->nullable();
```

Comment:
I know I specified placing it here but I was wrong.

Move it above measurement unit id, also in upgrade, form etc.